src/components/notifications/listing/TableView.js

```diff
@@ -0,0 +1,245 @@
+                ) : (
+                    <TableBody>
+                        {(!data || data.length === 0) && (
+                            <EmptyTable
+                                message={t("noNotifications")}
+                                numColumns={columnData.length}
+                            />
+                        )}
+                        {data?.length > 0 &&
+                            data.map((n) => {
+                                const isSelected = selected.includes(
+                                    n.message.id
+                                );
+
+                                const className = n.seen
+                                    ? null
+                                    : classes.unSeenNotificationBackground;
+
+                                return (
+                                    <TableRow
+                                        onClick={(event) =>
+                                            handleRowClick(event, n.message.id)
+                                        }
+                                        role="checkbox"
+                                        aria-checked={isSelected}
+                                        tabIndex={-1}
+                                        selected={isSelected}
+                                        hover
+                                        key={n.message.id}
+                                    >
+                                        <TableCell
+                                            className={className}
+                                            padding="checkbox"
+                                        >
+                                            <DECheckbox checked={isSelected} />
+                                        </TableCell>
+                                        <TableCell className={className}>
+                                            {
+                                                notificationCategory[
+                                                    n.type
+                                                        .replace(/\s/g, "_")
+                                                        .toLowerCase()
+                                                ]
+                                            }
+                                        </TableCell>
+                                        <Message
+                                            className={classnames(
+                                                classes.notification,
+                                                className
+                                            )}
+                                            message={n.message}
+                                            onMessageClicked={onMessageClicked}
+                                        />
+                                        <TableCell className={className}>
+                                            {formatDate(n.message.timestamp)}
```

Comment:
This date cell and the category cell should probably also contain a `Typography` since the message cell does.  It's hard to tell from the screenshots, but I do think the message cell font looks slightly different from these two and it may be due to this?